src/posts/2020-09-11-announcing-alchemy-conf.md

```diff
@@ -0,0 +1,118 @@
+---
+path: /posts/announcing-alchemy-conf/
+title: "Announcing Alchemy Conf"
+author: roberto-machado
+date: 2020-09-11
+tags:
+  - development
+  - conferences
+  - elixir
+intro: >
+  How important is the community for you? It's been four years since our last
+  conference. And now we're back.
+---
+
+How important is the community for you? I get asked this sometimes and
+inevitably end up reminiscing about Subvisual’s origins. At the time, there
+wasn’t a Subvisual, if I’m honest. No, that was still a few years down the road.
+There was a Group Buddies, however. A small Ruby and Rails consultancy firm. We
+were still taking our first steps and barely made by. This was in 2012, a year
+that, in retrospect, defined us. We were presented with the opportunity to
+support a small Ruby conference called Ruby Norte and decided to bite the
+bullet. Sponsorship was "only” 250€. The use of quotes is not an exaggeration
+since, at the time, when we stared at our bank account statement, we saw 200€
+staring back at us. The remaining 50€ were put out of our own pockets. I can
+never give a correct answer to the question of "how important the community is,”
+but **I can look back and proudly say it has always been a part of us.**
+
+A lot more followed. **We started organizing several meetups** (Braga.rb,
+Braga.js, Braga.Design, Braga.Product, Braga BEAM), **helped others by hosting
```

Comment:
```suggestion
Braga.js, Braga.Design, Braga.Product, Braga BEAM), helped others by hosting
```

Reply:
The end of this highlight comes after "Mirror Conf", a few lines below. [Deploy preview here](https://deploy-preview-385--subvisual.netlify.app/blog/posts/announcing-alchemy-conf/). Is it too much though?